sprokit/src/tools/pipe_to_dot.h

```diff
@@ -28,36 +28,33 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef SPROKIT_TOOLS_TOOL_USAGE_H
-#define SPROKIT_TOOLS_TOOL_USAGE_H
+#ifndef KWIVER_TOOL_PIPE_CONFIG_H
+#define KWIVER_TOOL_PIPE_TO_DOT_H
 
-#include <sprokit/tools/sprokit_tools_export.h>
-#include <vital/vital_config.h>
+#include <tools/kwiver_applet.h>
 
-#include <sprokit/pipeline_util/pipeline_builder.h>
-#include <sprokit/pipeline/scheduler_factory.h>
+#include <string>
+#include <vector>
 
-#include <boost/program_options/options_description.hpp>
-#include <boost/program_options/variables_map.hpp>
+namespace sprokit {
+namespace tools {
 
-namespace sprokit
+class pipe_to_dot
+  : public kwiver::tools::kwiver_applet
 {
+public:
+  pipe_to_dot();
 
-[[noreturn]] SPROKIT_TOOLS_EXPORT void tool_usage(int ret, boost::program_options::options_description const& options);
-SPROKIT_TOOLS_EXPORT void tool_version_message();
+  virtual int run() override;
+  virtual void add_command_options() override;
 
-SPROKIT_TOOLS_EXPORT boost::program_options::options_description tool_common_options();
+  static constexpr char const* name = "pipe_to_dot";
```

Comment:
I think we want to use a consistent *verb-noun* pattern for applet names.  Maybe it should be called `graph-pipeline` (or `graph-pipe`)?  If that's too obscure, in this case, I think `pipe-to-dot` is okay even if it doesn't exactly match the pattern.